contracts/finance/VestingConfidential.sol

```diff
@@ -0,0 +1,167 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I was expecting the _confidential_ version of the `VestingWallet.sol` contract, could you explain why having a design pretty far from that?
Having said, what I like in this approach is the multi beneficiary/recipient that `VestingWallet` doesn't have if I'm not wrong,